package.json

```diff
@@ -30,5 +30,8 @@
     "nx": "14.6.4",
     "prettier": "^2.7.1",
     "typescript": "^4.8.2"
+  },
+  "dependencies": {
+    "react-countdown-circle-timer": "^3.0.9"
```

Comment:
these should be in `apps/frontend`'s `package.json`

Reply:
Updated!